src/_main_/utils/common.py

```diff
@@ -163,12 +163,26 @@ def resize_image(img, options={}):
     new_img = cv2.resize(img, dsize=size, interpolation = cv2.INTER_AREA)
     return new_img
 
-
 def _common_name(s):
   return (' '.join(s.split('_'))).title()
 
 def validate_fields(args, checklist):
   for field in checklist:
     if field not in args:
       return False, CustomMassenergizeError(f"You are missing: {_common_name(field)}")
-  return True, None
\ No newline at end of file
+  return True, None
+
+def get_cookie(request, key):
+  cookie = request.COOKIES.get(key)
+  if cookie and len(cookie) > 0:
+      return cookie
+  else:
+      return None 
+
+def set_cookie(response, key, value): # TODO
+  print(f"----- set_cookie: {response}")
+  # set cookie on response before sending
+  # cookie expiration set to 1yr
+  MAX_AGE = 31536000
+
+  response.set_cookie(key, value, MAX_AGE, samesite='Strict')
```

Comment:
Note the recent change made to auth.py:
response.set_cookie("token", secure=True, value=token, max_age=MAX_AGE, samesite='None')

This was needed for the custom subdomain sites which are not massenergize.org.

Reply:
Okay, this should be an easy enough fix, though that function is not used at the moment. I let the front end handle the device cookies, per Sam's suggestion.